### DISCUSSION TITLE
Option to remove 'Donate' Buttons after donating via PayPal

### DISCUSSION THREAD

**Discussion by Globespy** [Rem0o/FanControl.Releases#1035] [Q&A] [answered]
I know this is a minor issue, but nonetheless would be useful for some.
I donated $10 to the dev via PayPal, it would be nice to have some registration key/code that can remove the multiple 'Donate' options every time I open the main menu.
Thanks for reading.

**Comment by Rem0o** [comment, marked as answer]
Such a petty thing would require much more infrastructure and effort that you would think.
Would require almost a complete license system hooked to donations. And I don't think any of you want a license system in FanControl, and neither do I.  I never intended to give benefits to donation/paid users because I don't want the "paid user syndrome" to start creeping in on the software

I've drawn the line for donation prompts to a single popup you can opt-out, and buttons that are out of view in app menus. I use my own app myself, without alteration, and I think it is fairly unobtrusive for the user.

Unless this raises an issue for a lot more people, I would keep it this way.

**Reply by Globespy** [reply]
Thank you, I did not realize it would require so much effort.
